Sources/Sentry/SentryClient.m

```diff
@@ -102,6 +109,7 @@ - (BOOL)checkSampleRate:(NSNumber *)sampleRate {
 - (SentryEvent *_Nullable)prepareEvent:(SentryEvent *)event
                              withScope:(SentryScope *)scope {
     NSParameterAssert(event);
+    NSLog(@"prepareEvent");
```

Comment:
Please remove, we don't want any `NSLog` statements since you can't control them.

Reply:
Done